carla_ground_truth_integration/SubsystemControllerParams.yaml

```diff
@@ -32,7 +32,7 @@ message:
       subsystem_namespace: /message
       required_subsystem_nodes: ['']
       unmanaged_required_nodes:
-        - /hardware_interface/dsrc_driver_node
+        - /hardware_interface/v2x_ros_driver_node
```

Comment:
I suggest that we remove any updates to the carla* and xil* configs in this PR since the CDA Sim deployment doesn't actually use the V2X ROS Driver (it didn't use the DSRC Driver either, but it looks like this was accidentally included). 
The appropriate fix would be to remove this line completely, but there may be some testing that would be required to confirm everything still functions properly. If the CDA Sim project(s) decide to clean up their existing configurations then we can handle this work under a separate PR.